project/DistributionPackage.scala

```diff
@@ -136,11 +136,7 @@ object DistributionPackage {
     )
     val os = System.getProperty("os.name")
     val isMac = os.startsWith("Mac")
-    val dir = if (isMac) {
-      "target/rust/x86_64-apple-darwin/debug/"
-    } else {
-      "target/rust/debug/"
-    }
+    val dir = "target/rust/debug/"
```

Comment:
This  value should be provided from the outside, hard-coding this is probably a bad idea